routing.go

```diff
@@ -19,20 +15,20 @@ import (
 )
 
 type proxyRouting struct {
-	kuboRPC    []string
```

Comment:
@hacdias  I've been thinking about the deployment story and  we can't remove `kuboRPC` from `proxyRouting` just yet. We need to have a release with both `kuboRPC` AND `gatewayURLs` so our infrastructure can migrate without breaking IPNS on Rhea.

Mind restoring `kuboRPC` here and using it when `KUBO_RPC_URL` is set and  `IPNS_RECORD_GATEWAY` is not set?
(If explicit `IPNS_RECORD_GATEWAY` is set, it should take preference over `KUBO_RPC_URL`).